src/pages/ProductList/components/GoodList/Heart.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { AiOutlineHeart } from 'react-icons/ai';
+import { AiFillHeart } from 'react-icons/ai';
+
+export const Heart = () => {
+  const [liked, setLiked] = useState(false);
+  return (
+    <div>
+      {liked ? (
+        <AiFillHeart
+          onClick={() => setLiked(!liked)}
+          className="heart"
+          liked={liked}
+          setLiked={setLiked}
+        />
+      ) : (
+        <AiOutlineHeart
+          onClick={() => setLiked(!liked)}
+          className="heart"
+          liked={liked}
+          setLiked={setLiked}
+        />
```

Comment:
똑같은 함수가 반복해서 사용되는데, 이벤트에서 함수를 분리하고 같은 함수를 등록해주는 방식으로 수정 해 보세요



```suggestion

const switchLikes = ()=>{setLiked(prev=>!prev)}
return(
...
        <AiFillHeart
          onClick={switchLikes}
          className="heart"
          liked={liked}
          setLiked={setLiked}
        />
      ) : (
        <AiOutlineHeart
          onClick={switchLikes}
          className="heart"
          liked={liked}
          setLiked={setLiked}
        />
```

Reply:
반영했습니다.